### PR TITLE
Bounty 3 ( editBounty/editSubmission/sucess )

### PR DESCRIPTION
**Pull request by Holybasil**
**What kind of change does this PR introduce?**

Add mobile support for create bounty/submission.

**Did you add tests for your changes?**
No.

**Does this PR introduce a breaking change?**
No.

**What needs to be documented once your changes are merged?**
Nothing.

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-bounty/77)

### CODE REVIEW

**Comment by Holybasil** [comment]
周末有空的话会再看看Modal/MobileModal消失的动画

File: src/components/Picker/picker.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.rmc-picker,.rmc-multi-picker {
```
Comment (21paradox) [inline src/components/Picker/picker.css]: styled componnet?

File: src/components/MobileModal/index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  close = () => {
+    const { closeModal } = this.props;
+    closeModal();
+  };
+
+  render() {
+    const { show, children } = this.props;
+    if (show) {
+      document.addEventListener('touchmove', MobileModal.preventDefault, { passive: false });
```
[the remark this answers is not in the document]
Comment (Holybasil) [reply src/components/MobileModal/index.js]: 可以放生命周期里 配合this.props.show 这里可以跟动画效果一起考虑

File: src/pages/Bounty/editbounty.js [hunk 3]
```diff
@@ -320,7 +383,7 @@ class EditBounty extends Component {
           content={i18nTxtAsync('bounty.faq')}
           title={i18nTxt('Bounty Example')}
           wrapStyle={{
-            width: '400px',
+            width: window.screen.width >= 600 ? '400px' : '100%',
```
[the remark this answers is not in the document]
Comment (Holybasil) [reply src/pages/Bounty/editbounty.js]: 有两处遗漏未使用isMobile 已改

File: src/utils/device.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        // np不做转换。1np就是1px 100np就是100px
+        break;
+      case 'dp':
+      default:
+        // 注意这里375,设计稿是按照375进行设计的。
+        // deviceWidth为屏幕的宽度,iphone 5/SE为320 iphone 6/7/8为375
+        myCount = (myCount / 375) * window.screen.width;
+    }
+
+    if (!allowMiniPixel() && myCount < 1) {
```
[the remark this answers is not in the document]
Comment (Holybasil) [reply src/utils/device.js]: 也可以将unitParser缓存

**Comment by Holybasil** [comment]
> > _[Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-bounty/77)_ status: 8 of 45 files reviewed, 7 unresolved discussions (waiting on @Holybasil and @yqrashawn)
> 
> _[src/index.html, line 8 at r7](https://reviewable.io/reviews/conflux-chain/conflux-bounty/77#-Lqxws-s4J9fx7sm45SR:-Lqxws-s4J9fx7sm45SS:b-8xwgrx) ([raw file](https://github.com/conflux-chain/conflux-bounty/blob/8c42c38f60fbca75ad9a73135a73f1cbbb97357b/src/index.html#L8)):_
> 
> > ```
> >     <title>Conflux Bounty</title>
> >     
> >     <meta name="viewport" content="width=device-width, initial-scale=1">
> > ```
> 
> 建议加-–上user-scalable=no，不让移动端缩放
> 
> _[src/pages/Bounty/commonStyle.js, line 58 at r8](https://reviewable.io/reviews/conflux-chain/conflux-bounty/77#-Lqy1B_t41Dy_ims8Gw1:-Lqy1B_t41Dy_ims8Gw2:b-5hzb56) ([raw file](https://github.com/conflux-chain/conflux-bounty/blob/9e6c372d7ca9dbc1b3fe6f7a8b0ba5060dfa3f9b/src/pages/Bounty/commonStyle.js#L58)):_
> 
> > ```js
> >     display: inline-block;
> >     margin-top: 6px;
> >     margin-right: 10px;
> > ```
> 
> 这里删掉是不是会影响PC端
> 
> _[src/utils/index.js, line 651 at r8](https://reviewable.io/reviews/conflux-chain/conflux-bounty/77#-Lqy2DHU-w0wD9-ABrYg:-Lqy2DHU-w0wD9-ABrYh:b-x1rlih) ([raw file](https://github.com/conflux-chain/conflux-bounty/blob/d35f364fb60283acf3b6e9b99e283fe27c990acf/src/utils/index.js#L651)):_
> 
> > ```js
> >             style={{
> >               maxWidth: '100%',
> >               // marginTop: 10,
> > ```
> 
> 这里注释掉是为啥呢

1.已加。
2.父级元素已使用grid统一布局（PC和移动端），所以删掉了针对子元素的布局。
3.第一个附件的image没有与右侧元素对齐，所以删掉了，现在已取消注释。

**Pull request closed without merging**